### ISSUE TITLE
No suitable driver found for jdbc:redshift

### ISSUE THREAD

**Issue by redraw** [metabase/metabase#6350]
I recently upgraded to the latest Metabase version (0.26.2), using Docker. I stopped and removed my running container, pull the new docker image (`docker pull metabase`), and started a new container with,
```
docker run -d -p 3000:3000 -v /home/ubuntu/metabase:/tmp -e MB_DB_FILE=/tmp/metabase.db --name metabase metabase/metabase
```

Everything was preserved, but Amazon Redshift database doesn't seem to work. i.e when saving settings in Admin Panel -> Databases -> [our Redshift DB], it throws
```
java.sql.SQLException: No suitable driver found for jdbc:redshift://db.panoply.io:5439/[db-name]?OpenSourceSubProtocolOverride=false
```

I tried changing the backend to Postgres, and it works. There's just a warning,
```
Nov 08, 2017 2:12:00 PM org.postgresql.jdbc.PgConnection <init>
WARNING: Unsupported Server Version: 8.0.2
```

However, some queries I had in Metabase doesn't work, and I'm not sure if that's because I've selected  Postgres backend in Metabase instead of Amazon Redshift. An error example is,
```
ERROR: Specified types or functions (one per INFO message) not supported on Redshift tables.
```

**Comment by jornh** [comment]
Hit the same problem with the Redshift driver. We finally found out that it was due to mapping to the `/tmp` folder similar to what you have above in your docker run command. The reason seemed to be that with the Redshift driver apparently some `hsperfdata` files will be written to `/tmp`.

Try moving to a volume mapping location similar to what the install guide suggests under https://metabase.com/docs/latest/operations-guide/running-metabase-on-docker.html#mounting-a-mapped-file-storage-volume 

We ended up using a slightly different solution which I don't have the exact details of right now - but can share if the above doesn't work out for you.

And, yes you're right, the updated Postgres driver which was introduced to support Postgres 10 is not compatible with Redshift so the Metabase team introduced the new Redshift driver.
